app/models/etmp/EtmpRegistrationRequest.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.etmp
+
+import models.requests.RegistrationRequest
+import models.BankDetails
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.domain.Vrn
+
+case class EtmpRegistrationRequest(
+                             vrn: Vrn,
+                             tradingNames: Seq[EtmpTradingNames],
+                             schemeDetails: EtmpSchemeDetails,
+                             bankDetails: BankDetails
+                           ) {
+}
+
+object EtmpRegistrationRequest {
+
+  def fromRegistrationRequest(registration: RegistrationRequest): EtmpRegistrationRequest = {
```

Comment:
can we add some specific unit tests for this? Ensuring the converted model matches what we expect

Reply:
Added unit test for model conversion.